packages/perseus/src/widgets/grapher/util.tsx

```diff
@@ -1,661 +1,35 @@
 import {point as kpoint} from "@khanacademy/kmath";
+import {GrapherUtil} from "@khanacademy/perseus-core";
```

Comment:
In #2089 (which will likely be closed without merging) I felt like I went too far moving utilities out of `perseus`. So this time I _only_ moved what I had to move to get the Grapher scorer onto `perseus-score`.